app/uk/gov/hmrc/plasticpackagingtax/registration/views/check_primary_contact_details_page.scala.html

```diff
@@ -36,7 +36,7 @@
 @extractAddress(address: Address) = @{
     Seq(
         Some(address.addressLine1),
-        address.addressLine2,
+        Some(address.addressLine2),
```

Comment:
Again, out of curiosity, why are we wrapping the mandatory fields in Optionals?

Reply:
Does the `flatten` call fail if 1+ of the contents of the `Seq` are not `Optionals`?